controllers/courses.js

```diff
@@ -770,7 +770,7 @@ router.get('/:courseId/', async (req, res, next) => {
 	}
 });
 
-router.patch('/:courseId', (req, res, next) => {
+router.patch('/:courseId', async (req, res, next) => {
```

Comment:
when using async here, we must try/catch the whole body for catch cou may use `catch(next)`, please return res.foo